toolkit/do.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
why is this necessary? Once the packages are built and published I think this will no longer be useful and it probably should not be on the main branch